build.sbt

```diff
@@ -33,9 +31,7 @@ developers := List(
   )
 )
 
-ThisBuild / tlSonatypeUseLegacyHost    := true
-ThisBuild / githubWorkflowJavaVersions := Seq(JavaSpec(Temurin, "8"))
```

Comment:
This is not necessary because the default `githubWorkflowJavaVersions` is Temurin JDK 8 (see https://github.com/sbt/sbt-github-actions/blob/main/src/main/scala/sbtghactions/GenerativePlugin.scala#L581)